lib/charms/loki_k8s/v0/loki_push_api.py

```diff
@@ -866,20 +868,20 @@ def _from_dir(self, dir_path: Path, recursive: bool) -> List[dict]:
 
         return alert_groups
 
-    def add_path(self, path: str, *, recursive: bool = False):
+    def add_path(self, path_str: str, *, recursive: bool = False):
```

Comment:
Just curious what was wrong with `path`?

Reply:
It was being immediately rewritten with `path = Path(path)`, which the linter didn't like